src/components/UI/ClientTable/ClientTable.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
After looking into the different bugs with `DataTable` from `react-data-table-component`, I started from scratch and started using Material-UI table components instead, just to see if that would be a valid workaround. Now, in the `ServerTable.tsx` file, is where that component lies, with fully-functioning server-side pagination.

However, I found that `DataTable` was really easy to use and get up and running, so I decided to keep the component wrapping the DataTable around, as I think it is more advantageous than the Material UI table when it comes to client-side pagination or not having to worry about passing in so many props + maintaining their states. I named this one `ClientTable` and the other `ServerTable` for reference, and I think each one has its own advantages/disadvantages with respect to certain use cases.